src/core/agent/services/connectionService.ts

```diff
@@ -313,29 +310,28 @@ class ConnectionService extends AgentService {
   }
 
   @OnlineOnly
-  async resolveOobi(url: string, isMultisigRelated?: boolean): Promise<any> {
+  async resolveOobi(url: string, waitForCompletion?: boolean): Promise<any> {
```

Comment:
I would still prefer `waitForCompletion = false`. Mixing `undefined` in with booleans works in this case but could lead to future bugs if you explicitly check for `=== false` etc